packages/cardano-services/test/Program/loadHttpServer.test.ts

```diff
@@ -45,9 +47,10 @@ describe('loadHttpServer', () => {
         options: {
           cardanoNodeConfigPath,
           dbConnectionString,
+          dbQueriesCacheDuration,
           ogmiosUrl: new URL(ogmiosConnection.address.webSocket)
         },
-        serviceNames: [ServiceNames.StakePool, ServiceNames.TxSubmit, ServiceNames.NetworkInfo]
+        serviceNames: [ServiceNames.StakePool, ServiceNames.TxSubmit]
```

Comment:
This one is the problematic test where `getaddrinfo ENOTFOUND database.server.com` is thrown, caused by service [healthCheck](https://github.com/input-output-hk/cardano-js-sdk/blob/12ce8b4c18d1df087fb5d1a7295ebe569aa6d188/packages/cardano-services/src/NetworkInfo/NetworkInfoHttpService.ts#L34-L36). Same behavior if we add [ChainHistoryHttpService](https://github.com/input-output-hk/cardano-js-sdk/blob/64aa7aeff061aa2cf9bc6196347f6cf5b9c7f6be/packages/cardano-services/src/ChainHistory/ChainHistoryHttpService.ts#L16) in `serviceNames` - the containerised db is not accesssible during the service creation when there is `db.query` health check prior inisialisation. Looks like a raise condition, this is the only failing test if all available services are added here.

Reply:
cc @iadmytro